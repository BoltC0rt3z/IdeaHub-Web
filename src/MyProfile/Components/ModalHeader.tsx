import React from 'react';
import { Row, Col } from 'antd';
import { Button } from '../../Components';

import styles from './ModalHeader.module.scss';

const ModalHeader: React.FC = () => {
  return (
    <Row type="flex" justify="space-between">
      <Col>
        <Row type="flex">
          <Col>
            <div className={styles.closeIcon}>&#10005;</div>
          </Col>
          <Col className={styles.headerText}>Edit profile</Col>
        </Row>
      </Col>
      <Col>
        <Button type="primary" size="small">
          Save
        </Button>
      </Col>
    </Row>
  );
};

export default ModalHeader;
