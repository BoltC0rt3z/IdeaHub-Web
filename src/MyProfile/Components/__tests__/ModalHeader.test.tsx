import React from 'react';
import { mount, ReactWrapper } from 'enzyme';

import ModalHeader from '../ModalHeader';

describe('Tests for modal header', () => {
  const wrapper: ReactWrapper = mount(<ModalHeader />);

  it('Should render the header without crushing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
