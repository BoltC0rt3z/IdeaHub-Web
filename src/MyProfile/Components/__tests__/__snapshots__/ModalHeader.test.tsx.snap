// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for modal header Should render the header without crushing 1`] = `ReactWrapper {}`;
